src/System.CodeDom/src/System.CodeDom.csproj

```diff
@@ -134,21 +134,8 @@
   <ItemGroup Condition="'$(TargetGroup)' == 'net461'">
     <TargetingPackReference Include="System" />
```

Comment:
I suspect this is dead code now and this doesn't actually build for netfx but @tarekgh might need to clean that up as part of the netfx configuration work.